keyless/circuit-tests/src/sha.rs

```diff
@@ -0,0 +1,65 @@
+use rand::{RngCore, thread_rng};
+use sha2::Digest;
+use aptos_keyless_common::input_processing::circuit_input_signals::CircuitInputSignals;
+use aptos_keyless_common::input_processing::config::CircuitPaddingConfig;
+use crate::TestCircuitHandle;
+
+fn sha256_padding(message: &[u8]) -> Vec<u8> {
```

Comment:
For this fn, I think it would be good to use the code from the prover service [here](https://github.com/aptos-labs/prover-service/blob/main/src/input_processing/sha.rs) instead of re-implementing it.

Reply:
hmm but that function takes `&str` instead of bytes...

IMO circuit-test and prover service should share a function that works with bytes (so the current one), and prover service should have its own wrapper that works with &str?

Reply:
As part of testing the circuit bug you found, I imported this code from the prover server and changed it to work on `&[u8]`. So you should be able to use it without problem now